cookiecutter/{{cookiecutter.organization}}-richie-site-factory/docker-compose.yml

```diff
@@ -4,13 +4,13 @@ services:
   db:
     image: postgres:9.6
     environment:
-      - POSTGRES_DB=richie_${RICHIE_SITE}
+      - POSTGRES_DB=richie_${RICHIE_SITE:?}
```

Comment:
Not quit sure to understand the purpose of this syntax.

Reply:
it's to make it required. Otherwise it evaluates to an empty string and we start creating directories where we don't want to! :sweat_smile:

Reply:
Oh ! From my side if RICHIE_SITE is not set, docker compose returns an error. So with or without `:?`, I have the same behaviour.